data/types/bolus/pen/pen.go

```diff
@@ -0,0 +1,63 @@
+package pen
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/data/types/bolus"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	SubType = "pen" // TODO: Rename Type to "bolus/normal"; remove SubType
+
+	NormalMaximum = 100.0
+	NormalMinimum = 0.0
```

Comment:
```suggestion
	normalMaximum = 100.0
	normalMinimum = 0.0
```
We do not need to export theses values.

Reply:
again, tests are missing